.github/workflows/docker-build-and-publish.yml

```
@@ -0,0 +1,32 @@
+name: Deploy Images to Github Container registry
+
+on:
+  push:
+
+jobs:
+  push-store-image:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Add tag for new version
+        id: semanticVersion
+        uses: anothrNick/github-tag-action@1.67.0
+        env:
+          DEFAULT_BUMP: patch
+          DRY_RUN: github.ref != 'refs/heads/main'
+          GITHUB_TOKEN: ${{ secrets.GIT_TOKEN}}
```

```suggestion

      - name: Extract metadata (tags, labels) for Docker
        id: meta
        uses: docker/metadata-action@v5
        with:
          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
```

I'm not entirely sure what the step that you added does @philter87. Does it make the tag automatically? If so, I feel that should be a separate action maybe?

As described in https://docs.github.com/en/actions/publishing-packages/publishing-docker-images, using this to get information about the tags and labels we (manually?) add. To be honest, I don't completely know what meta data output this gives. How are tags and labels defined in Docker?